docs/other_vscode.md

```diff
@@ -167,7 +167,7 @@ You'll need to perform some modifications to the file above in order to target y
 * `"BMPGDBSerialPort"`: The `COM` port under Windows, or the `/dev/...` path for Linux/macOS.
 * `"svdFile"`: _[Optional]_ The path to the SVD file that defines the register layout for the MCU -- the appropriate file can be downloaded from the [cmsis-svd repository](https://github.com/posborne/cmsis-svd/tree/master/data/STMicro)
 * `"device"`: The name of the MCU, which matches the `<name>` tag at the top of the downloaded `svd` file.
-* `"armToolchainPath"`: _[Optional]_ The path to the ARM toolchain installation location on Windows -- under normal circumstances Linux/macOS will auto-detect this correctly and will not need to be specified. Windows is special, which is why it has its own section.
+* `"armToolchainPath"`: _[Optional]_ The path to the ARM toolchain installation location on Windows -- under normal circumstances Linux/macOS will auto-detect this correctly and will not need to be specified. Windows is special, which is why it has its own section. You can get a copy of the GNU ARM toolchain for Windows from [developer.arm.com](https://developer.arm.com/tools-and-software/open-source-software/developer-tools/gnu-toolchain/gnu-rm/downloads).
```

Comment:
MSYS2 has an arm-none-eabi-gdb package, just FYI

Reply:
Does it need to be executed from within the MSYS2 environment? VSCode executes externally... unsure if it'll work.

Reply:
The entire set of ARM toolchain packages are compiled as native binaries (this is the `mingw-w64-x86_64` or `i686` prefix), so they don't depend on [msys-2.0.dll](https://www.msys2.org/wiki/How-does-MSYS2-differ-from-Cygwin/), unlike eg. pacman and git. Even so, I seem to be able to run MSYS packages from the command prompt just fine too.

Reply:
Interesting... guess I should set up a Windows env for QMK and try it out. Should simplify the docs significantly if we can leverage it, especially considering it'll match the compiler build the firmwares get built with.

Reply:
For macOS, actually, the gdb binary isn't present, and has to be downloaded manually.  

Though, correct me if I'm wrong, but that is something that we can fix/add, I think.

Reply:
There is `arm-none-eabi-gdb` in the core tap.

Reply:
Ah, last I cheched, I swear it wasn't.  Also, it's 13.2, and not sure if the version matching gcc matters here.

Reply:
Added notes to point MSYS2 at `gdb-multiarch.exe`. Rest should be sweet, now.